packages/appstore/src/components/wallet-content/wallet-transfer-block.tsx

```diff
@@ -11,11 +11,16 @@ type TProps = {
 };
 
 const WalletTransferBlock = observer(({ wallet_account }: TProps) => {
-    const { traders_hub, ui } = useStore();
+    const { traders_hub, ui, client } = useStore();
     const { setIsWalletModalVisible } = ui;
     const { setWalletModalActiveWalletID, setWalletModalActiveTab } = traders_hub;
+    const { accounts } = client;
```

Comment:
Note: I'm not using `usePlatformAccounts` hook because it returns all REAL accounts (`usePlatformRealAccounts` hook) and just 1st DEMO account (`usePlatformDemoAccount` hook), but we have at least 2 DEMO accounts (VRTC and VRW) and I don't know which one will be 1st. In this case I've used `accounts` from `client` store. Let me know what do you think, maybe we need to create one more hook or modify existed?

Reply:
How about using `useAuthorize` and getting the account_list. Then you can filter it here

Reply:
@thisyahlen-deriv @farzin-deriv `useAuthorize` hook uses `accounts` from `client` store too, gets token from there and fetching. In result we have response without `balance`, and I need to make one more request (balance). I'm not sure it's the best solution here. I see just 2 cases: 1. create one more hook with 2 requests (authorize and balance); 2. use `client` store. What do you think?

Reply:
I think its better if we can just add balance to the `useAuthorize`

Reply:
Plus you can update the `useAuthorize` hook to use `client_accounts` from local storage to get the token if you want to remove stores dependancy

Reply:
@thisyahlen-deriv I'm not sure it's a good idea to add balance to the `useAuthorize`, because I suppose hooks must be as small as possible and have a single responsibility. About removing store dependancy, I'm not sure we need to avoid store dependancy, I just asked is it ok to use `client` store or we need to avoid using store? As for me we can use `client` store. @farzin-deriv what do you think?

Reply:
Guys I'm not sure what's the plan :(
I have added some store free and entity base hooks a couple of weeks ago but I'm just keeping them in the stash and waiting to get some clarification on what we want to do first.

<img width="445" alt="image" src="https://github.com/binary-com/deriv-app/assets/72082844/ab660a8c-8f72-416e-ba24-1b5058d24abb">

Reply:
@farzin-deriv Thanks for your answer. In this case I will leave using store in this PR and we'll refactor it later when we make a decision